kolibri/utils/cli.py

```diff
@@ -235,9 +235,24 @@ def start(port=None, daemon=True):
 
     if not daemon:
         logger.info("Running 'kolibri start' in foreground...")
+
     else:
         logger.info("Running 'kolibri start' as daemon (system service)")
 
+    __, urls = server.get_urls(listen_port=port)
+    if not urls:
+        logger.error(
+            "Could not detect an IP address that Kolibri binds to, but try "
+            "opening up the following addresses:\n")
+        urls = [
+            "http://{}:{}".format(ip, port) for ip in ("0.0.0.0", "127.0.0.1")
```

Comment:
You're probably right - there's some more definitions of it here: https://en.wikipedia.org/wiki/0.0.0.0

Notice the text above should make it clear to the user that the IPs may not work.

What I was mostly judging from was simply that 0.0.0.0 resolves to localhost, you can also use http://0:8080 as a short.. but dunno if it's just Firefox that resolves it and redirects.

Fell free to call the shots! But the intention is to make this decision as irrelevant as possible :)